test/runners/python_spec.js

```diff
@@ -225,3 +225,102 @@ describe('Output format commands', function() {
     });
   }
 });
+
+describe('Fixed and new features', function() {
```

Comment:
Can you also add the test for the Unicode fix (#508, #545)?

Reply:
> could be solvable by providing the source charset on the first line, which in turn requires some edits in the runner code

Do you mean adding `# coding: utf-8`?

Reply:
Yes. But since we're going for multiple files approach, we'll need it in each file (at least preloaded, solution, and fixture).

Reply:
If we go multiple files, can users just add them when needed?
This is currently impossible because we concatenate them and the magic comment is only valid if it's in the first or the second line.

Reply:
Since we'll want to prepend something to the code when needed (just like in your `unittest` fix), I'd just add it to every file I create. At least it won't break existing things (because utf-8 is a superset of ascii).